packages/ember-runtime/lib/mixins/action_handler.js

```diff
@@ -209,6 +209,20 @@ var ActionHandler = Mixin.create({
                    ') does not have a `send` method', typeof target.send === 'function');
       target.send(...arguments);
     }
+  },
+
+  /**
+   Check if controller has any subscriptions for named event.
+
+   @method hasAction
+   @param {String} actionName The action to trigger
+   @public
+   */
+  hasAction(actionName) {
+    if (this._actions && actionName) {
+      return this._actions.hasOwnProperty(actionName);
```

Comment:
[looks like](https://github.com/emberjs/ember.js/blob/master/packages/ember-runtime/lib/mixins/action_handler.js#L202-L203) to attain symmetry with dispatch, this line should be `actionName in this._actions`